tdrs-frontend/src/assets/Button.scss

```diff
@@ -22,3 +22,7 @@
     box-shadow: inset 0 0 0 2px #1a4480;
     color: #1a4480;
 }
+
+#button-anchor {
```

Comment:
I used the button instead of link.href. The button underlined the text to I added this css to make it look the same.